core/src/main/java/me/leoko/advancedban/manager/UUIDManager.java

```diff
@@ -208,7 +209,10 @@ public String getNameFromUUID(String uuid, boolean forceInitial) {
     private String askAPI(String url, String name, String key) throws IOException {
     	MethodInterface mi = mi();
         name = name.toLowerCase();
-        HttpURLConnection request = (HttpURLConnection) new URL(url.replaceAll("%NAME%", name).replaceAll("%TIMESTAMP%", new Date().getTime() + "")).openConnection();
+
```

Comment:
This makes the code more complicated without much measurable gain (the slow performance of the http connection will dwarf any regex by comparison). Also, you can use String#replace and you don't need regex at all